### ISSUE TITLE
In High Contrast Aquatic mode, unselected stars are not properly visible present in 'Rate your experience!' card.

### ISSUE THREAD

**Issue by Kirank915** [microsoft/BotFramework-WebChat#5103]
### Is it an issue related to Adaptive Cards?

No.

### What is the PWD impact?

Color sensitive users who depend on high contrast modes are getting impacted, as In High Contrast Aquatic theme mode, unselected stars are not properly visible. This will impact the user's desired task.

### What browsers and screen readers do this issue affect?

Windows: Chrome with NVDA

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Forced colors (high contrast mode)

### What is the public URL for the website?

[Request Contributors - POC (sharepoint.com)](https://microsoft.sharepoint.com/teams/MCAPSAcademy/SitePages/Request-a-Contributor---POC.aspx?xsdata=MDV8MDF8ZGJoYWt0YUBtaWNyb3NvZnQuY29tfGUzYWVmOTc1NDViMTQzYTg4Y2ZhMDhkYmM1YmQ5YjI3fDcyZjk4OGJmODZmMTQxYWY5MWFiMmQ3Y2QwMTFkYjQ3fDF8MHw2MzgzMjExOTAzODMzODc3NDF8VW5rbm93bnxUV0ZwYkdac2IzZDhleUpXSWpvaU1DNHdMakF3TURBaUxDSlFJam9pVjJsdU16SWlMQ0pCVGlJNklrMWhhV3dpTENKWFZDSTZNbjA9fDMwMDB8fHw%3D&sdata=SSs2MzJWZWN5M2JyTVZGeWp4c0VjbkJlNHIvZEpueXNreC9IZktzSkMxTT0%3D&OR=Teams-HL&CT=1696587959401&clickparams=eyJBcHBOYW1lIjoiVGVhbXMtRGVza3RvcCIsIkFwcFZlcnNpb24iOiIyNy8yMzA5MTQxNzUwNSIsIkhhc0ZlZGVyYXRlZFVzZXIiOmZhbHNlfQ%3D%3D)

### How to reproduce the issue?

Step 1: Open the above URL in New Edge/Chrome browsers.
Step 2: Request Contributors - POC Home page will appear. 
Step 3: Navigate to the 'Got questions?' in the page now Ask Iris Now! link is invoked it, chat box appeared. 
Step 4: Navigate to the 'What skills are needed to be a contributor?' control and invoke it now 'Did that answer your question or solve your problem?' question will opened now navigate to the 'No, create a Support Ticket; and invoke it. now 'Select a tool, workstream or team from the dropdown' appeared Navigate to the 'Continue' Button invoke it Next 'Submit' will be appeared and invoke the Submit Button now 'Is there anything else that I can help you with?' question will be appeared now 'No' Button invoke it Now 'Rate your experience!'  will appear.
Step 5: Now navigate to the unselected stars. 
Step 6: Verify whether In High Contrast Aquatic theme mode, unselected stars are not properly visible or not.

### What do you expect?

In High Contrast Aquatic theme mode, Unselected stars should be properly visible.

### What actually happened?

In High Contrast Aquatic theme mode, unselected stars are not properly visible present in 'Rate your experience!' card.

### Do you have any screenshots or recordings to repro the issue?

<img width="1856" alt="MAS 4 3 1-In High Contrast Aquatic theme mode, unselected stars are not properly visible" src="https://github.com/microsoft/BotFramework-WebChat/assets/117266902/c38e91fd-37d8-482b-a5f8-8070f4f732d0">

### MAS reference

https://aka.ms/MAS4.3.1

**Comment by Kirank915** [comment]
#A11yMAS;#A11ySev3;#MAS4.3.1;#HCL;#Accessibility;#FeatureBotframeworkwebchat-Apr23;

**Comment by Kirank915** [comment]
Bug copied from MCAPS: [ISSUE 1767692](https://dev.azure.com/worldwidelearning/WWL%20Accessibility/_workitems/edit/1767692)

**Comment by compulim** [comment]
This is related to bot author. The bot author is using images that does not display properly in high contrast mode.

**Comment by Kirank915** [comment]
As per the above comment, this issue is not under bot framework team. Hence closed the bug and updated to partner team (WWL)
